static/styles/zulip.scss

```diff
@@ -2757,3 +2757,23 @@ div.topic_edit_spinner .loading_indicator_spinner {
         top: 5%;
     }
 }
+#compose-container {
+    &.error-shake {
+        animation: shake 0.5s ease-in-out 0s 1;
+    }
+}
+
+@keyframes shake {
+    0% {
+        margin-left: 0rem;
```

Comment:
Using the CSS `transform: translateX(...)` will result in better performance than using `margin`. See https://stackoverflow.com/a/53892597/4083221.

Reply:
Yes, that's quite fair. Will update.